main.css

```diff
@@ -344,3 +346,356 @@
+    flex-direction: row;
+    justify-content: space-between;
+    align-items:center;
+    background-color: #3e3c3c;
+    color: #fff;
+    padding:10px;
+    width: 100%;
+  }
+
+  .header-section .title {
+    text-decoration: none;
+    color: #fff;
+    margin-left: 20px;
+  }
+
+  .header-section .menu {
+    display: flex;
+    flex-direction: row;
+    margin-right: 20px;
+  }
+
+  .header-section .menu .menu-links {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .header-section .menu .menu-links a {
+    text-decoration: none;
+    list-style: none;
+    margin-right: 14px;
+    color: white;
+  }
+
+  .header-section .toggle-icon {
+    display: none;
+  }
+
+  .hero-section {
+    background: #1c1a19 url(./images/heroDesktopBg.png);
+    background-repeat: no-repeat;
+    background-size: 380px;
+    background-position-x: 930px;
+    background-position-y: -44px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    width: 100%;
+    height: 440px;
+    gap: 50px; 
+  }
+
+  .hero-section .hero-text {
+    color: #fff;
+    width: 400px;
+    height: 123px;
+  }
+  
+  .hero-section .hero-title h1 {
+    color: #f55800;
+    font-size: 72px;
+    font-family: 'Crete regular', serif;
+  }
+
+  .hero-section .hero-title h3 {
+    font-family: 'Inter', sans-serif;
+  }
+
+  .hero-section .hero-text .hero-desc {
+    font-family: 'Inter regular', sans-serif;
+    padding-top: 12px;
+    line-height: 20px;
+    font-size: 12px;
+  }
+
+  .hero-section .hero-social-links {
+    margin-left:16px;
+    list-style: none;
+    padding-top: 100px;
+  }
+
+  .hero-section .hero-social-links img {
+    width: 60%;
+    margin-top: 10px;
+  }
+
+  .works-section .Work-section-title {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    margin: 50px 50px 50px 50px;
+  }
+
+  .works-section .Work-section-title .vector {
+    width: 65%;
+    height: 3px;
+    margin-top: 12px;
+  }
+
+  .works-section .Work-section-title h2{
+    color: #091E42;
+  } 
+
+  .works-section .works-text {
+    display: flex;
+    flex-direction: row;
+    gap: 14px;
+    margin: 0 50px 50px 50px;
+  }
+
+  .works-section .works-text .work-desc-links {
+    list-style: none;
+    display: flex;
+    gap: 14px;
+    margin: 14px 0 14px 0;
+  }
+
+  .works-section .works-text .work-desc-links  li {
+    border: #3e3c3c 1px solid;
+    padding: 5px;
+  }
+  .works-section .works-text .work-desc-links a {
+    text-decoration: none;
+  }
+  .works-section .works-links h3 {
+    margin-bottom: 14px;
+    color: #091E42;
+  }
+  .works-section .works-links .work-section-button {
+    padding: 8px;
+    background:#f55800;
+    border: none;
+  }
+
+  .projects-section {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    margin: 50px 50px 50px 50px;
+    gap: 14px;
+  }
+
+  .projects-section .project-item {
+    
+    height:300px;
+    padding: 50px 10px 10px 10px;
+    color:#fff;
+  }
```

Comment:
- [ ] Hey, your project card is missing the hover effect. When the project is hovered on with the mouse the description and dark opacity should disappear and the `See Project button` appears at the bottom of the image which should take the user to the Live Demo of your portfolio or to the GitHub repository. 
Please, kindly implement this feature.